src/components/Select/Select.tsx

```diff
@@ -191,20 +193,21 @@ export function Select(props: SelectProps): React.ReactElement {
 
     if (
       !loading &&
-      props.allowCreatingNewItems &&
       items.filter(item => item.label.toString().toLowerCase().includes(query.toLowerCase())).length === 0
     )
       return (
         <React.Fragment>
           <div className={css.noResultsFound}>Nothing Found</div>
-          <Button
-            intent="primary"
-            minimal
-            text={query}
-            icon="plus"
-            className={css.createNewItemButton}
-            onClick={handleClick as React.MouseEventHandler<Element>}
-          />
+          {props.allowCreatingNewItems && (
+            <Button
+              intent="primary"
+              minimal
+              text={query}
+              icon="plus"
+              className={css.createNewItemButton}
+              onClick={handleClick as React.MouseEventHandler<Element>}
+            />
+          )}
```

Comment:
If `props.allowCreatingNewItems` is false, you want to show this Button. Earlier it was not shown. Is this expected?

Reply:
Thanks for the review.
If `props.allowCreatingNewItems` is `false`, then also the button will not be shown.

Old flow:
If false - nothing will be rendered. No dropdown or anything. User does not know if what they typed in the query is there or not.
If true - "Nothing found" text with "Add" button

New flow:
if false - Atleast "Nothing found" text will be rendered in the dropdown. User will know there is no such item.
If true - "Nothing found" text with "Add" button


So it is safe. Better UX.